install/docker/cws-image/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM oraclelinux:8
 
 RUN yum update -y && \
-    yum install -y mariadb-server java-1.8.0-openjdk java-1.8.0-openjdk-devel && \
+    yum install -y mariadb-server java-1.8.0-openjdk java-1.8.0-openjdk-devel rsync which && \
```

Comment:
take out which.  we don't really need that.  let's keep it clean.
There's another command that works like which that I think is already there.

Reply:
Never mind.  it's needed